src/java/org/apache/cassandra/cql3/selection/SelectorFactories.java

```diff
@@ -46,6 +46,11 @@ final class SelectorFactories implements Iterable<Selector.Factory>
      */
     private boolean containsWritetimeFactory;
 
+    /**
+     * <code>true</code> if one of the factory creates maxWritetime selectors.
```

Comment:
```suggestion
     * {@code true} if one of the factories creates maxWritetime selectors.
```

Reply:
`<code>true</code>` was used to be consistent with others. There are many such occurrences in this file.  I can change the nearby ones.